src/axolotl/utils/config.py

```diff
@@ -527,6 +527,22 @@ def validate_config(cfg):
     if cfg.fsdp and "bnb" in cfg.optimizer:
         raise ValueError(f"FSDP not compatible with {cfg.optimizer}")
 
+    if cfg.datasets:
+        for idx, ds_cfg in enumerate(cfg.datasets):
+            if (
+                ds_cfg.type
+                in [
+                    "ultra_apply_chatml",
+                    "toxic_apply_chatml",
+                    "chatml.intel",
+                    "chatml.argilla",
```

Comment:
```suggestion
```

Aren't these current formatters? I don't think we should expect the user to receive a warning if they use these two.

Reply:
chatml.intel is used here https://github.com/OpenAccess-AI-Collective/axolotl/blob/5787e1a23f61be093fc26067f8a05342fbdfbe1a/tests/e2e/test_dpo.py#L46 and the test seams to work. So thought they where supported but might miss something :) chatml.argilla is mention here https://github.com/OpenAccess-AI-Collective/axolotl/blob/5787e1a23f61be093fc26067f8a05342fbdfbe1a/docs/rlhf.md?plain=1#L29

Reply:
Right, if the chatml.* types.work, then we shouldn't we warning here? Or am I misunderstanding something?

Reply:
I don't fully understand what you're saying. My intention was to suggest that we should issue a warning or raise a ValueError if a conversation format is added, but there's already a type in place that formats the data. This situation arises, for example, with ultra_apply_chatml (see the example in the comment below). Here's another example where the conversation format is determined by the type: https://huggingface.co/cognitivecomputations/dolphin-2.6-mistral-7b-dpo/blob/5c32e515f3d79beefc110e8a07c3671269a0f5ab/configs/dolphin-dpo.yml#L14